pyseq.py

```diff
@@ -66,9 +66,12 @@
 # regex for matching format directives
 format_re = re.compile(r'%(?P<pad>\d+)?(?P<var>\w+)')
 
+# character to join explicit frame ranges on
+range_join = os.environ.get("PYSEQ_SEP", " ")
```

Comment:
Is this a common need? What would drive this need? Genuinely curious. 

If this is something we need to support, then I think a clearer environment variable name would be useful. PYSEQ_SEP is too generic.